src/data/faq.json

```
@@ -883,6 +883,20 @@
                             "This screen is shown on iPhones running an iOS version between iOS 13.5 and (including) iOS 13.6.1. On these iOS Versions, the necessary version 2 of the Exposure Notification Framework is not available. Therefore you need to update to iOS 13.7 or newer, to continue to use the Corona-Warn-App. More information can be found in <a href='#ios135' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry."
                         ]
                     },
+                    {
+                        "title": ["Why does the CWA not scan a QR code?",
+                                "  Why does the QR code scanner not respond?"

```

```suggestion
                                "Why does the QR code scanner not respond?"
```
remove leading space